### DISCUSSION TITLE
Import from mariaDB on RDS requires REPLICATION CLIENT

### DISCUSSION THREAD

**Discussion by sandy-garrido** [planetscale/discussion#254] [Bug Reports] [answered]
I have tried everything, even when I think I have this permission, it still doesn't recognise that I have it.

I give myself 0 permissions and I get [REPLICATION CLIENT, REPLICATION SLAVE, PROCESS]. I then give myself PROCESS and save it'll reduce it to [REPLICATION CLIENT, REPLICATION SLAVE]. I then give myself  REPLICATION SLAVE then left with [REPLICATION CLIENT] but even when I give myself this REPLICATION CLIENT.

I still get this error.. No matter what I do. I've follow the steps in the tutorial, though it still fails.

**Comment by jgreet** [comment, marked as answer]
Unfortunately our importer is only partially compatible with MariaDB. Assuming you're coming from MariaDB 10.5 or higher this is probably the cause:

https://mariadb.com/docs/reference/mdb/privileges/REPLICATION_CLIENT/

>Starting in ES10.5 and CS10.5 BINLOG MONITOR is a synonym for REPLICATION CLIENT but the capabilities have changed.

As a workaround we can turn off the Connection Test that occurs prior to starting the import, but that could mask other problems. [Open a ticket](https://support.planetscale.com/hc/en-us/requests/new) if you'd like to try that.

**Reply by sandy-garrido** [reply]
Thanks JGreet! I managed to get a ticket in, and have a response already similar to your message here. Though not just tried to see if it works or not. If it doesn't then at least I know why now so that's this answered here! Hopefully help other people in future! 👍
